pvm/ts/src/engine/actions/baseAction.test.ts

```
@@ -58,13 +58,12 @@
 
     describe("verify", () => {
         describe("game state validation", () => {
-            it("should throw error if game is in showdown", () => {
-                jest.spyOn(game, "currentRound", "get").mockReturnValue(TexasHoldemRound.SHOWDOWN);
-
-                expect(() => action.verify(player)).toThrow("Hand has ended.");
+            it.only("should throw error if player is not active", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)